src/components/App/App.js

```diff
@@ -18,21 +21,41 @@ function App() {
       .catch(error => setError(`Sorry, something went wrong. Please try again later.`))
       console.log(error)
   }, [])
+
+  const handleClick = () => {
+    getRandomQuote()
+      .then(data => {
+        setQuote({...data, isFavorite: false})
+      })
+  }
+
+  const handleFavorite = (event) => { 
```

Comment:
I think these would be better off living in the component they are called in, and passing down the state updates and functionalities to the components, but for a small app you can get away with it. If you kept up with the storing of these handle functions in the top level of App.js it would become unruly and bloated as the project grew.